graphics/wxWidgets-3.2/Portfile

```diff
@@ -50,7 +50,8 @@ depends_lib-append  port:jpeg \
 depends_run         port:wxWidgets-common \
                     port:wxWidgets_select
 
-patchfiles          patch-configure.diff
+patchfiles          patch-configure.diff \
```

Comment:
Is `patch-configure` still required? From what I understand it's for the wxGTK build variant; surely the "native" build shouldn't find OpenGL from $prefix?

Reply:
I lost this, sorry! seems to works fine without. removed.
thanks for the good catch.